### ISSUE TITLE
[Tarea 1] CSV de output

### ISSUE THREAD

**Issue by bgsamilla** [IIC2523/2020-2-Tarea1#12]
Hola!

Es necesario que al csv de output le agregemos los nombres de las columnas como primera linea o header?
O solo con los datos basta?

Saludos!

**Comment by FranLucchini** [comment]
No es necesario, con los datos basta.

**Comment by Arcoirisky** [comment]
El CSV final es el que queda luego de haber realizado las N operaciones?
osea, las N operaciones no son independientes entre si, sino que trabajan sobre la anterior ¿No?

**Comment by FranLucchini** [comment]
Hola, 

No entiendo muy bien tu duda, a qué te refieres con N operaciones? Ojalá me lo puedas aclarar.
En caso de que sea esto: cada ejecución del programa es sólo una operación, por ejemplo un Select, por lo tanto sería sólo 1 CSV.

**Comment by Arcoirisky** [comment]
Eso mismo!

**Comment by FranLucchini** [comment]
Ah, entonces la duda era porque pensabas que había que hacer todas las operaciones en una misma ejecución?

**Comment by Arcoirisky** [comment]
Claro, pensaba por un momento que los N threads significaban N operaciones dependientes de la anterior

Por ejemplo, si hacía (1) SELECT (2)GROUP (3)SELECT 

Entonces la entrada de (2) sería la salida de (1) y no la base original.

Pero ya entendí que no es así xD y me alegra porque tendría que cambiar varias cosas
